test/mitmproxy/test_version.py

```diff
@@ -17,20 +17,18 @@ def test_version(capsys):
 
 
 def test_get_version():
-    version.VERSION = "3.0.0rc2"
-
```

Comment:
I removed this hard coded version since when testing my generated TS file for the BACKEND_CONSTS there was a mismatch between `"3.0.0rc2"` and `"11.0.0 dev"`